components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,741 @@
+
+momentum:
+
+$$
+\frac{D \rho {\bf u}_{3D} }{D t} \equiv
+\frac{\partial \rho {\bf u}_{3D}}{\partial t}
+ + \nabla_{3D} \cdot \left( \rho {\bf u}_{3D} \otimes {\bf u}_{3D}  \right)
+  = - \nabla_{3D} p
++ \rho {\bf D}^u + \rho {\bf F}^u
+$$ (continuous-momentum)
+
+mass:
+
+$$
+\frac{D \rho}{D t} \equiv
+\frac{\partial \rho }{\partial t}
+ + \nabla_{3D} \cdot \left( \rho  {\bf u}_{3D}  \right)
+= 0
+$$ (continuous-mass)
+
+tracers:
+
+$$
+\frac{D \rho \varphi }{D t} \equiv
+\frac{\partial \rho \varphi}{\partial t}
+ + \nabla_{3D} \cdot \left( \rho \varphi {\bf u}_{3D}  \right)
+= D^\varphi + Q^\varphi
+$$ (continuous-tracer)
+
+Here we have express the following terms as a general operators, with examples of specific forms provided below: the dissipation ${\bf D}^u$, momentum forcing ${\bf F}^u$, tracer diffusion $D^\varphi$, and tracer sources and sinks $Q^\varphi$. The momentum equation can be rewritten using the product rule on $\rho {\bf u}$, mass conservation, and dividing by $\rho$, as:
+
+$$
+\frac{D {\bf u}_{3D} }{D t} \equiv
+\frac{\partial {\bf u}_{3D}}{\partial t}
+ + \nabla_{3D} \cdot \left( {\bf u}_{3D} \otimes {\bf u}_{3D}  \right)
+  = - \frac{1}{\rho} \nabla_{3D} p
++ {\bf D}^u + {\bf F}^u
+$$ (continuous-momentum-rho)
+
+Geophysical fluids such as the ocean and atmosphere are rotating and stratified, and horizontal velocities are orders of magnitude larger than vertical velocities. It is therefore convenient to separate the horizontal and vertical as ${\bf u}_{3D} = \left( {\bf u}, w \right)$ and $\nabla_{3D} = \left( \nabla_z, d/dz \right)$ where $z$ is the vertical direction in a local coordinate frame on the Earth's surface and $w$ is the vertical velocity. The $z$ subscript on $\nabla_z$ is to remind us that this is the true horizontal gradient (perpendicular to $z$), as opposed to gradients within tilted layers used in the following section. The gravitational force $g {\bf k}$ is added in the vertical. The rotating frame of reference results in the Coriolis force $f {\bf k} \times {\bf u} \equiv f {\bf u}^\perp$, where $f$ is the Coriolis parameter and ${\bf u}^\perp$ is the horizontal velocity rotated $90^o$ from $\bf u$. See any textbook in the [References](#references) for a full derivation.
+
+The horizontal parts of the advective term may be replaced with the vector identity
+
+$$
+\nabla_z \cdot \left( {\bf u} \otimes {\bf u}  \right) =
+\boldsymbol{u} \cdot \nabla_z \boldsymbol{u} &= (\nabla_z \times \boldsymbol{u}) \times \boldsymbol{u} + \nabla_z \frac{|\boldsymbol{u}|^2}{2} \\
+&= \{\boldsymbol{k} \cdot (\nabla_z \times \boldsymbol{u})\} \boldsymbol{k} \times \boldsymbol{u} + \nabla_z \frac{|\boldsymbol{u}|^2}{2} \\
+&= \zeta \boldsymbol{u}^{\perp} + \nabla_z K.
+$$ (advection-identity)
```

Comment:
to get this set above to render in visual studio I needed this form 
```
$$
\begin{aligned}
\nabla_z \cdot \left( {\bf u} \otimes {\bf u}  \right) &=
\boldsymbol{u} \cdot \nabla_z \boldsymbol{u} \\
&= (\nabla_z \times \boldsymbol{u}) \times \boldsymbol{u} + \nabla_z \frac{|\boldsymbol{u}|^2}{2} \\
&= \{\boldsymbol{k} \cdot (\nabla_z \times \boldsymbol{u})\} \boldsymbol{k} \times \boldsymbol{u} + \nabla_z \frac{|\boldsymbol{u}|^2}{2} \\
&= \zeta \boldsymbol{u}^{\perp} + \nabla_z K.
\end{aligned}
$$ (advection-identity)
```

Reply:
Thanks! I didn't know that.